CMSgov/eAPD
# Email State Admin from State Dashboard when Affiliation is Pending

Resolves #2771

### This pull request is ready to merge when...

- [x] Tests have been updated (and all tests are passing)

### This feature is done when...

- [x] QA has approved the experience

### Steps to verify this PR:

* Create an Okta User: https://test.reg.idm.cms.gov/registration.html?appid=eapd
* Login
* Select 'Alaska'
* Verify that 'State Administrator' link is present with `href='mailto:...'`
* Login into the em@il user account and verify the new user(s) requested for Alaska is in pending deliveries.

Repeat steps above with another user, a different state (e.g. - 'Alabama') and verify 'State Administrator' is present with 'href=mailto:CMS-EAPD@cms.hhs.gov' (no state admin assigned, yet).
